[2020-11-25 15:56:32] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Access to undeclared static property: Api::$res" at C:\wamp64\www\VAP-final\override\classes\order\OrderHistory.php line 585 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Access to undeclared static property: Api::$res at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderHistory.php:585)"} []
[2020-11-25 15:58:08] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Access to undeclared static property: Api::$res" at C:\wamp64\www\VAP-final\override\classes\order\OrderHistory.php line 585 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Access to undeclared static property: Api::$res at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderHistory.php:585)"} []
[2020-12-02 11:35:21] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedMethodException: "Attempted to call an undefined method named "datefr" of class "OrderHistory"." at C:\wamp64\www\VAP-final\override\classes\order\OrderHistory.php line 622 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedMethodException(code: 0): Attempted to call an undefined method named \"datefr\" of class \"OrderHistory\". at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderHistory.php:622)"} []
[2020-12-02 11:37:20] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedMethodException: "Attempted to call an undefined method named "datefr" of class "OrderHistory"." at C:\wamp64\www\VAP-final\override\classes\order\OrderHistory.php line 623 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedMethodException(code: 0): Attempted to call an undefined method named \"datefr\" of class \"OrderHistory\". at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderHistory.php:623)"} []
[2020-12-02 16:22:50] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "C:\wamp64\www\VAP-final\var\cache\prod\app5D44.tmp" to "C:\wamp64\www\VAP-final\var\cache\prod/appProdProjectContainerUrlMatcher.php"." at C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\Filesystem\Filesystem.php line 289 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"C:\\wamp64\\www\\VAP-final\\var\\cache\\prod\\app5D44.tmp\" to \"C:\\wamp64\\www\\VAP-final\\var\\cache\\prod/appProdProjectContainerUrlMatcher.php\". at C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Filesystem\\Filesystem.php:289)"} []
[2020-12-03 15:00:29] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Parse error: syntax error, unexpected ')'" at C:\wamp64\www\VAP-final\override\classes\order\OrderHistory.php line 585 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Parse error: syntax error, unexpected ')' at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderHistory.php:585)"} []
[2020-12-03 15:00:37] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Parse error: syntax error, unexpected ')'" at C:\wamp64\www\VAP-final\override\classes\order\OrderHistory.php line 585 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Parse error: syntax error, unexpected ')' at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderHistory.php:585)"} []
[2020-12-03 15:06:34] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Cannot use object of type stdClass as array" at C:\wamp64\www\VAP-final\override\controllers\Api.php line 146 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Cannot use object of type stdClass as array at C:\\wamp64\\www\\VAP-final\\override\\controllers\\Api.php:146)"} []
[2020-12-04 09:43:55] request.INFO: Matched route "admin_module_manage". {"route":"admin_module_manage","route_parameters":{"category":null,"keyword":null,"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::manageAction","_legacy_controller":"AdminModulesManage","_legacy_link":["AdminModulesManage","AdminModulesSf"],"_route":"admin_module_manage"},"request_uri":"http://localhost/VAP-final/admin038euyw7y/index.php/improve/modules/manage?_token=QCy5XCZB-V1hulsC5G_jcl7_7RniAOOb4aqBF9p3NeY","method":"GET"} []
[2020-12-04 09:43:55] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2020-12-04 09:43:55] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"m.djellouli@cmp-paris.com"} []
[2020-12-04 09:43:56] app.ERROR: The addon with id 29992 does not have name. [] []
[2020-12-04 11:29:55] request.INFO: Matched route "admin_module_manage". {"route":"admin_module_manage","route_parameters":{"category":null,"keyword":null,"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::manageAction","_legacy_controller":"AdminModulesManage","_legacy_link":["AdminModulesManage","AdminModulesSf"],"_route":"admin_module_manage"},"request_uri":"http://localhost/VAP-final/admin038euyw7y/index.php/improve/modules/manage?_token=QCy5XCZB-V1hulsC5G_jcl7_7RniAOOb4aqBF9p3NeY","method":"GET"} []
[2020-12-04 11:29:55] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2020-12-04 11:29:55] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"m.djellouli@cmp-paris.com"} []
[2020-12-04 11:29:55] app.ERROR: The addon with id 29992 does not have name. [] []
[2020-12-04 11:30:17] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2020-12-04 14:00:18] request.INFO: Matched route "admin_module_manage". {"route":"admin_module_manage","route_parameters":{"category":null,"keyword":null,"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::manageAction","_legacy_controller":"AdminModulesManage","_legacy_link":["AdminModulesManage","AdminModulesSf"],"_route":"admin_module_manage"},"request_uri":"http://localhost/VAP-final/admin038euyw7y/index.php/improve/modules/manage?_token=QCy5XCZB-V1hulsC5G_jcl7_7RniAOOb4aqBF9p3NeY","method":"GET"} []
[2020-12-04 14:00:18] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2020-12-04 14:00:18] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"m.djellouli@cmp-paris.com"} []
[2020-12-04 14:00:18] app.ERROR: The addon with id 29992 does not have name. [] []
[2020-12-14 10:29:40] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Failed to start the session because headers have already been sent by "C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\VarDumper\Dumper\AbstractDumper.php" at line 181." at C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php line 141 {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\VarDumper\\Dumper\\AbstractDumper.php\" at line 181. at C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:141)"} []
[2020-12-14 10:29:51] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Failed to start the session because headers have already been sent by "C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\VarDumper\Dumper\AbstractDumper.php" at line 181." at C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php line 141 {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\VarDumper\\Dumper\\AbstractDumper.php\" at line 181. at C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:141)"} []
[2020-12-14 10:56:51] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Failed to start the session because headers have already been sent by "C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\VarDumper\Dumper\AbstractDumper.php" at line 181." at C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php line 141 {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\VarDumper\\Dumper\\AbstractDumper.php\" at line 181. at C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:141)"} []
[2020-12-14 11:07:08] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Failed to start the session because headers have already been sent by "C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\VarDumper\Dumper\AbstractDumper.php" at line 181." at C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php line 141 {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\VarDumper\\Dumper\\AbstractDumper.php\" at line 181. at C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:141)"} []
[2020-12-14 11:08:35] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Failed to start the session because headers have already been sent by "C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\VarDumper\Dumper\AbstractDumper.php" at line 181." at C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php line 141 {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\VarDumper\\Dumper\\AbstractDumper.php\" at line 181. at C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:141)"} []
[2020-12-14 11:20:13] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 130 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:130)"} []
[2020-12-14 11:20:19] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 130 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:130)"} []
[2020-12-14 11:20:31] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 130 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:130)"} []
[2020-12-14 11:22:21] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 132 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:132)"} []
[2020-12-14 11:22:23] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 132 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:132)"} []
[2020-12-14 11:23:15] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 132 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:132)"} []
[2020-12-14 11:26:22] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 132 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:132)"} []
[2020-12-14 11:34:12] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Failed to start the session because headers have already been sent by "C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\VarDumper\Dumper\AbstractDumper.php" at line 181." at C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php line 141 {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\VarDumper\\Dumper\\AbstractDumper.php\" at line 181. at C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:141)"} []
[2020-12-14 11:35:08] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Failed to start the session because headers have already been sent by "C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\VarDumper\Dumper\AbstractDumper.php" at line 181." at C:\wamp64\www\VAP-final\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage.php line 141 {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\VarDumper\\Dumper\\AbstractDumper.php\" at line 181. at C:\\wamp64\\www\\VAP-final\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:141)"} []
[2020-12-14 13:47:24] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 132 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:132)"} []
[2020-12-14 13:47:45] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 132 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:132)"} []
[2020-12-14 13:47:46] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 132 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:132)"} []
[2020-12-14 13:48:47] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 132 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:132)"} []
[2020-12-14 13:52:19] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 132 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:132)"} []
[2020-12-14 13:54:30] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 133 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:133)"} []
[2020-12-14 13:54:32] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 133 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:133)"} []
[2020-12-14 13:54:33] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 133 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:133)"} []
[2020-12-14 13:55:13] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 134 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:134)"} []
[2020-12-14 13:55:22] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "connectionApi" from the global namespace." at C:\wamp64\www\VAP-final\override\classes\order\OrderState.php line 134 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"connectionApi\" from the global namespace. at C:\\wamp64\\www\\VAP-final\\override\\classes\\order\\OrderState.php:134)"} []
[2020-12-16 12:42:54] request.INFO: Matched route "admin_module_manage_action". {"route":"admin_module_manage_action","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::moduleAction","action":"upgrade","module_name":"simpleimportproduct","_route":"admin_module_manage_action"},"request_uri":"http://localhost/VAP-final/admin038euyw7y/index.php/improve/modules/manage/action/upgrade/simpleimportproduct?_token=GmMgOGmHZWBw6aBRqZ6yZnPagWmZ7eRQa8WMgrhJnM0","method":"POST"} []
[2020-12-16 12:42:54] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2020-12-16 12:42:54] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"m.djellouli@cmp-paris.com"} []
[2020-12-16 12:42:54] app.ERROR: Exception retournée par le module simpleimportproduct pour upgrade. Error sent by Addons. You may need to be logged. [] []
[2020-12-16 12:42:54] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2020-12-17 15:02:40] request.INFO: Matched route "admin_mbo_catalog_module". {"route":"admin_mbo_catalog_module","route_parameters":{"_controller":"mbo.controller.modules.catalog:indexAction","_legacy_controller":"AdminPsMboModule","_legacy_link":"AdminPsMboModule","_route":"admin_mbo_catalog_module"},"request_uri":"http://localhost/VAP-final/admin038euyw7y/index.php/modules/addons/modules/catalog?_token=_t1P9hJydE8TcvdEA7KQ2zhwJ7JvpvGsMBDjtL6Jjuc","method":"GET"} []
[2020-12-17 15:02:40] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2020-12-17 15:02:40] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"m.djellouli@cmp-paris.com"} []
[2020-12-17 15:02:40] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'")." at C:\wamp64\www\VAP-final\src\PrestaShopBundle\Resources\views\Admin\layout.html.twig line 34 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'\"). at C:\\wamp64\\www\\VAP-final\\src\\PrestaShopBundle\\Resources\\views\\Admin\\layout.html.twig:34, SmartyException(code: 0): Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl' at C:\\wamp64\\www\\VAP-final\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_template.php:195)"} []
[2020-12-17 15:02:40] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2020-12-17 15:03:03] request.INFO: Matched route "admin_mbo_catalog_module". {"route":"admin_mbo_catalog_module","route_parameters":{"_controller":"mbo.controller.modules.catalog:indexAction","_legacy_controller":"AdminPsMboModule","_legacy_link":"AdminPsMboModule","_route":"admin_mbo_catalog_module"},"request_uri":"http://localhost/VAP-final/admin038euyw7y/index.php/modules/addons/modules/catalog?_token=_t1P9hJydE8TcvdEA7KQ2zhwJ7JvpvGsMBDjtL6Jjuc","method":"GET"} []
[2020-12-17 15:03:03] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2020-12-17 15:03:03] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"m.djellouli@cmp-paris.com"} []
[2020-12-17 15:03:03] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'")." at C:\wamp64\www\VAP-final\src\PrestaShopBundle\Resources\views\Admin\layout.html.twig line 34 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'\"). at C:\\wamp64\\www\\VAP-final\\src\\PrestaShopBundle\\Resources\\views\\Admin\\layout.html.twig:34, SmartyException(code: 0): Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl' at C:\\wamp64\\www\\VAP-final\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_template.php:195)"} []
[2020-12-17 15:03:03] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2020-12-17 15:03:05] request.INFO: Matched route "admin_mbo_catalog_module". {"route":"admin_mbo_catalog_module","route_parameters":{"_controller":"mbo.controller.modules.catalog:indexAction","_legacy_controller":"AdminPsMboModule","_legacy_link":"AdminPsMboModule","_route":"admin_mbo_catalog_module"},"request_uri":"http://localhost/VAP-final/admin038euyw7y/index.php/modules/addons/modules/catalog?_token=_t1P9hJydE8TcvdEA7KQ2zhwJ7JvpvGsMBDjtL6Jjuc","method":"GET"} []
[2020-12-17 15:03:05] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2020-12-17 15:03:05] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"m.djellouli@cmp-paris.com"} []
[2020-12-17 15:03:05] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'")." at C:\wamp64\www\VAP-final\src\PrestaShopBundle\Resources\views\Admin\layout.html.twig line 34 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'\"). at C:\\wamp64\\www\\VAP-final\\src\\PrestaShopBundle\\Resources\\views\\Admin\\layout.html.twig:34, SmartyException(code: 0): Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl' at C:\\wamp64\\www\\VAP-final\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_template.php:195)"} []
[2020-12-17 15:03:05] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2020-12-17 15:03:07] request.INFO: Matched route "admin_mbo_catalog_module". {"route":"admin_mbo_catalog_module","route_parameters":{"_controller":"mbo.controller.modules.catalog:indexAction","_legacy_controller":"AdminPsMboModule","_legacy_link":"AdminPsMboModule","_route":"admin_mbo_catalog_module"},"request_uri":"http://localhost/VAP-final/admin038euyw7y/index.php/modules/addons/modules/catalog?_token=_t1P9hJydE8TcvdEA7KQ2zhwJ7JvpvGsMBDjtL6Jjuc","method":"GET"} []
[2020-12-17 15:03:07] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2020-12-17 15:03:07] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"m.djellouli@cmp-paris.com"} []
[2020-12-17 15:03:07] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'")." at C:\wamp64\www\VAP-final\src\PrestaShopBundle\Resources\views\Admin\layout.html.twig line 34 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'\"). at C:\\wamp64\\www\\VAP-final\\src\\PrestaShopBundle\\Resources\\views\\Admin\\layout.html.twig:34, SmartyException(code: 0): Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl' at C:\\wamp64\\www\\VAP-final\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_template.php:195)"} []
[2020-12-17 15:03:07] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2020-12-17 15:03:08] request.INFO: Matched route "admin_mbo_catalog_module". {"route":"admin_mbo_catalog_module","route_parameters":{"_controller":"mbo.controller.modules.catalog:indexAction","_legacy_controller":"AdminPsMboModule","_legacy_link":"AdminPsMboModule","_route":"admin_mbo_catalog_module"},"request_uri":"http://localhost/VAP-final/admin038euyw7y/index.php/modules/addons/modules/catalog?_token=_t1P9hJydE8TcvdEA7KQ2zhwJ7JvpvGsMBDjtL6Jjuc","method":"GET"} []
[2020-12-17 15:03:08] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2020-12-17 15:03:08] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"m.djellouli@cmp-paris.com"} []
[2020-12-17 15:03:09] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'")." at C:\wamp64\www\VAP-final\src\PrestaShopBundle\Resources\views\Admin\layout.html.twig line 34 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'\"). at C:\\wamp64\\www\\VAP-final\\src\\PrestaShopBundle\\Resources\\views\\Admin\\layout.html.twig:34, SmartyException(code: 0): Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl' at C:\\wamp64\\www\\VAP-final\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_template.php:195)"} []
[2020-12-17 15:03:09] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2020-12-17 15:03:14] request.CRITICAL: Uncaught PHP Exception SmartyException: "Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'" at C:\wamp64\www\VAP-final\vendor\smarty\smarty\libs\sysplugins\smarty_internal_template.php line 195 {"exception":"[object] (SmartyException(code: 0): Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl' at C:\\wamp64\\www\\VAP-final\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_template.php:195)"} []
[2020-12-17 15:03:30] request.CRITICAL: Uncaught PHP Exception SmartyException: "Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl'" at C:\wamp64\www\VAP-final\vendor\smarty\smarty\libs\sysplugins\smarty_internal_template.php line 195 {"exception":"[object] (SmartyException(code: 0): Unable to load template 'file:modules/Updatestatus/views/templates/hook/CommentProduct.tpl' at C:\\wamp64\\www\\VAP-final\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_template.php:195)"} []
[2020-12-17 15:15:47] request.INFO: Matched route "admin_module_manage_action". {"route":"admin_module_manage_action","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::moduleAction","action":"install","module_name":"commentproduct","_route":"admin_module_manage_action"},"request_uri":"http://localhost/VAP-final/admin038euyw7y/index.php/improve/modules/manage/action/install/commentproduct?_token=_t1P9hJydE8TcvdEA7KQ2zhwJ7JvpvGsMBDjtL6Jjuc","method":"POST"} []
[2020-12-17 15:15:47] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2020-12-17 15:15:47] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"m.djellouli@cmp-paris.com"} []
[2020-12-17 15:15:47] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: Cannot declare class CommentProduct, because the name is already in use" at C:\wamp64\www\VAP-final\modules\commentproduct\commentproduct.php line 0 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: Cannot declare class CommentProduct, because the name is already in use at C:\\wamp64\\www\\VAP-final\\modules\\commentproduct\\commentproduct.php:0)"} []
[2020-12-17 15:15:47] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
